ensime_shared/client.py

```diff
@@ -26,6 +25,12 @@
 else:
     from Queue import Queue
 
+six_path = os.path.join(os.path.dirname(__file__), 'six')
+sys.path.insert(0, six_path)
+websocket_path = os.path.join(os.path.dirname(__file__), 'websocket-client')
+sys.path.insert(0, websocket_path)
+import websocket
```

Comment:
I think it'd be nice to hide away all the `sys.path` hackery in one place at early initialization, so you can keep normal-looking imports in the commonly-edited files and probably avoid linter complaints, etc. The `__init__.py` in `ensime_shared` might work, or it might have to go to `autoload/ensime.vim.py` and `rplugin/python3/ensime.py` where there is already path hackery. There is some duplication in that case, maybe there's a way to eliminate it but I didn't find a good one yet.